cassandra/tablets.py

```diff
@@ -0,0 +1,48 @@
+import bisect
+from threading import Lock
+
+# Experimental, this interface and use may change
+class Tablet(object):
+    """
+    Represents a single ScyllaDB tablet.
+    """
+    last_token = 0
+    new_replicas = None
+    replicas = None
+    stage = ""
+
+    def __init__(self, last_token = 0, new_replicas = None, replicas = None, stage = ""):
+        self.last_token = last_token
+        self.new_replicas = new_replicas
+        self.replicas = replicas
+        self.stage = stage
+
+    # Experimental, this interface and use may change
+    @staticmethod
+    def _is_valid_tablet(row):
+        return row.get("replicas") is not None and len(row.get("replicas")) != 0 and row.get("table_name") is not None and row.get("keyspace_name") is not None
+
+    # Experimental, this interface and use may change
+    @staticmethod
+    def from_row(row):
+        if Tablet._is_valid_tablet(row):
+            tablet = Tablet(row.get("last_token"), row.get("new_replicas"), row.get("replicas"), row.get("stage"))
+            return tablet
+        return None
+
+# Experimental, this interface and use may change
+class Tablets(object):
+    _tablets = {}
+    _tablets_lock = Lock()
```

Comment:
And now you can remove the lock (and import) as it's no longer used

Reply:
done